src/components/Home/MainMenu.tsx

```diff
@@ -98,90 +99,95 @@ export const MainMenu: FC<{ filter?: string }> = ({ filter }) => {
+                                            setIcons(
+                                                icons.map(i =>
+                                                    i.id === buttonIcon.id
+                                                        ? { ...i, shown: true }
+                                                        : i
+                                                )
+                                            )
+                                        }}
+                                        buttonIcon={buttonIcon}
+                                        isDeleting={false}
+                                        key={"menu_add_icon" + buttonIcon.id}
+                                        inMenu
+                                    />
+                                ))}
+                            </View>
+                        ))}
+                    </View>
+                </ModalCustom>
+                {icons
+                    .filter(i => i.shown)
+                    .filter(
+                        i =>
+                            i.id === 9 ||
+                            (filter
+                                ? i.title
+                                      .toLowerCase()
+                                      .includes(filter.toLowerCase())
+                                : true)
+                    )
+                    .map(buttonIcon => (
+                        <MenuButton
+                            onPress={() => {
+                                if (isDeleting) setIsDeleting(false)
+                                if (buttonIcon.id === 9) setModalVisible(true)
+                                // TODO: actual navigation
+                                if (!isDeleting && buttonIcon.id !== 9)
+                                    navigate("Saluti", { defaultName: "Ciao" })
+                            }}
+                            onLongPress={() => {
+                                if (buttonIcon.id !== 9)
+                                    setIsDeleting(!isDeleting)
+                            }}
+                            buttonIcon={buttonIcon}
+                            isDeleting={isDeleting}
+                            onDelete={() => {
+                                const { id } = buttonIcon
+                                setIcons(
+                                    icons.map(i =>
+                                        i.id === id ? { ...i, shown: false } : i
+                                    )
                                 )
-                            )
-                        }}
-                        key={"menu_" + buttonIcon.id}
-                    />
-                ))}
-        </ScrollView>
+                            }}
+                            key={"menu_" + buttonIcon.id}
+                        />
+                    ))}
+            </ScrollView>
+            <HighlightsManager />
```

Comment:
maaa quindi qui teniamo l'`HighlightsManager` nel `MainMenu`?
non facciamo più in modo che in `Home` siano accanto? tipo
```tsx
                    <PoliSearchBar
                        onChange={searchKey => setSearch(searchKey)}
                    />
                    <MainMenu filter={search} />
                    <HighlightsManager />
```

Reply:
sisi mi ero poi dimenticato di spostarlo